tests/runtest.py

```diff
@@ -685,9 +682,14 @@ def run_single_case(case, flags, opts, arg):
     timeout = int(arg.timeout)
 
     origdir = os.getcwd()
+    case_num = case.split('_')[0][1:]
+    case_dir = os.path.abspath('test_%s' % case_num)
```

Comment:
I'd like to see the test name instead of the number.

Reply:
Hmm.. now I think that we can have the number.  Sorry for the noise and please keep the current code.